deployment/helm/node-feature-discovery/templates/prometheus.yaml

```diff
@@ -0,0 +1,73 @@
+{{- if .Values.metrics.enable }}
```

Comment:
Should we have `metrics.prometheus` or `metrics.prometheus.enable` for this? WDYT? Could be false by default if we want to (I'm not sure if we do 🧐)